processing/dataplotly_algorithms.py

```diff
@@ -49,6 +49,7 @@ class DataPlotlyProcessingPlot(QgisAlgorithm):
 
     INPUT = 'INPUT'
     PLOT_TYPE = 'PLOT_TYPE'
+    PLOT_TITLE = 'PLOT_TITLE'
     PLOT_TYPE_OPTIONS = ['bar', 'pie', 'scatter', 'histogram']
```

Comment:
maybe it is worth to keep the order of the plugin plot (scatter, bar, histogram, pie)